injector/disk_pressure.go

```diff
@@ -75,7 +84,7 @@ func (i *diskPressureInjector) GetDisruptionKind() types.DisruptionKindName {
 func (i *diskPressureInjector) Inject() error {
 	// add read throttle
 	if i.spec.Throttling.ReadBytesPerSec != nil {
-		if err := i.config.Cgroup.DiskThrottleRead(i.config.Informer.Major(), *i.spec.Throttling.ReadBytesPerSec); err != nil {
+		if err := i.config.Cgroup.Write("blkio", i.getThrottleFilename(diskPressureThrottleModeRead), i.formatThrottle(*i.spec.Throttling.ReadBytesPerSec, diskPressureThrottleModeRead)); err != nil {
```

Comment:
worth it having `"blkio"` as a const string?

Reply:
+1